boxes/templates/boxes/box_list.html

```diff
@@ -1,81 +1,189 @@
+      {% if page_obj.has_previous %}
+      <a href="?page={{ page_obj.previous_page_number }}&amp;display={{ display_status }}"> &lt; Previous</a>
+      {% endif %}
+
+      {% if paginator.num_pages > 1 %}
+      <span class="current">
+        Page {{ page_obj.number }} of {{ paginator.num_pages }}
+      </span>
+      {% endif %}
+
+      {% if page_obj.has_next %}
+      <a href="?page={{ page_obj.next_page_number }}&amp;display={{ display_status }}">Next &gt; </a>
+      {% endif %}
+    </span>
+  </div>
+</div>
+<div class="md-modal md-effect-1" id="modal-1">
+  <div class="md-content">
+    <div class="row center-xs center-md center-lg">
+      <div class="col-xs-12 col-sm-12 col-md-12">
+        <h3 class="text-dark title">Create New Box</h3>
+        <p class="smalltext"> For others to be able to send you encrypted content through the web page, you will need to generate "boxes". Each box has an associated link that you can copy and share with any given person, it can only be used once. After the expiration date, the form is locked and the link can no longer be used.</p>
+        <form action="{% url "boxes_create" %}" method="post" class="">
+          {% csrf_token %}
+
+          <p>
+            {{ form.name.errors }}
+            {{ form.name.label_tag }}
+            {{ form.name }}
+          </p>
+          <p>
+            {{ form.expires_at.errors }}
+            {{ form.expires_at.label_tag }}
+            {{ form.expires_at }}
+          </p>
+
+          <div class="row">
+            <div class="col-xs-6">
+              <p>
+                {{ form.never_expires.errors }}
+
+                {{ form.never_expires.label_tag }}
+
+                {{ form.never_expires }}
+              </p>
+            </div>
+            <div class="col-xs-6">
+              <p>
+                {{ form.max_messages.errors }}
+                {{ form.max_messages.label_tag }}
+                {{ form.max_messages }}
+              </p>
+
+
+            </div>
+          </div>
+          <p>
+            {{ form.description.errors }}
+            {{ form.description.label_tag }}
+            {{ form.description }}
+          </p>
+        </div>
+
+        <div class="row middle-xs center-xs">
+          <div class="col-xs-6">
+            <button type="submit" class="btn-default" value=""> Create</button>
+
+          </div>
+          <div class="col-xs-6 ">
+            <button type="button" class="md-close btn-default">Close</button>
+          </div>
+        </div>
+      </div>
+    </form>
+  </div>
 </div>
+
+<div class="md-overlay"></div>
 {% endblock content %}
 
 {% block scripts %}
-    <script src="{% static 'javascripts/box_list.js' %}"></script>
+
+<script src="{% static 'javascripts/box_list.js' %}"></script>
+
+
+<script type="text/javascript" src="{% static 'javascripts/jquery.datetimepicker.min.js' %}"></script>
+<script src="{% static 'javascripts/box_create.js' %}"></script>
+
+
+</section>
```

Comment:
This section shouldn't be ended inside the `script`'s block. Probably you should end it after line 176, before ending the `content` block.